haiku-apps/catkeyseditor/catkeyseditor-0.1.1.recipe

```diff
@@ -10,7 +10,10 @@ HOMEPAGE="https://github.com/puckipedia/BeLocalized"
 COPYRIGHT="2014-2015 Puck Meerburg"
 LICENSE="MIT"
 REVISION="1"
```

Comment:
> Hmm, I don't know if we should use a newer srcGitRev. I guess we could, but I'd recommend that we wait for a go from a project leader.

Since the recipe has the version v0.1.1 we should stick to CatKeysEditor's fixed release of v.0.1.1. There've been a few changes to the app that are very useful. I think only some minor localization is missing from the new bits. We could create another easy GCI task for that and ask Puck if he would create a ne release v0.1.2 for us. Then we can create a "new" recipe for that, probably in the same GCI task, because there are practically no changes in the recipe.